### PR TITLE
add classes relations

### PR DESCRIPTION
**Pull request by ReAliens**
In this pull request, I implemented the following:-
-  create rental, book, and classroom classes and create associations between them and person, teacher and student classes

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```
Comment (mikethreels) [inline book.rb]: - [ ] according to the requirements Getters and Setters for @rentals should be present

File: book.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
```
Comment (mikethreels) [inline book.rb:7]: - [ ] A method for adding rentals should be present, please have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more details.

File: person.rb [hunk 3]
```diff
@@ -1,7 +1,8 @@
 require_relative 'corrector'
+require_relative 'rental'
 
 class Person
-  attr_reader :id, :corrector
+  attr_reader :id, :corrector, :rentals
```
Comment (mikethreels) [inline person.rb]: - [ ] in this class Getters and Setters for @rentals should be present

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Ahmed, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ReAliens** [comment]
thanks for your feedback I will do this changes immediately

**Review by BrianSammit — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello @ReAliens, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 
- Great job working on the required changes 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ReAliens** [comment]
thanks for your great review :smiley:

**Pull request merged**